autorest/codegen/serializers/builder_serializer.py

```diff
@@ -699,6 +699,15 @@ def _create_body_parameter(
         body_kwarg_name = builder.request_builder.parameters.body_parameter.client_name
         if isinstance(body_param.type, BinaryType):
             retval.append(f"_{body_kwarg_name} = {body_param.client_name}")
+            if not body_param.default_content_type:
+                content_types = "'" + "', '".join(body_param.content_types) + "'"
```

Comment:
nit, it might be easier to write 
```python
content_types = ", ".join([f"'{content_type}'" for content_type in body_param.content_types])
```

Reply:
I think both are ok and it depends on developer's preference